cmd/incusd/daemon.go

```diff
@@ -2739,3 +2745,40 @@ func (d *Daemon) getOVS() (*ovs.VSwitch, error) {
 
 	return d.ovs, nil
 }
+
+func (d *Daemon) setupLinstor() error {
+	d.linstorMu.Lock()
+	defer d.linstorMu.Unlock()
+
+	// Clear any existing client
+	d.linstor = nil
+
+	// Get the Linstor controller connection string
+	controllerConnection := d.globalConfig.LinstorControllerConnection()
```

Comment:
Maybe we need to be more strict on this one. Setting a connection string without a scheme seems to break the client. So I guess we should enforce `://` to be present in the string.